assets/install_scripts/postinstall

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+set -e
+
+if [ -d "/Library/Application Support/Adobe/Common/Plug-ins/7.0/MediaCore/ntsc-rs.plugin" ]; then
+    echo "Signing After Effects plugin with ad-hoc signature..."
+    /usr/bin/codesign --force --deep --sign - "/Library/Application Support/Adobe/Common/Plug-ins/7.0/MediaCore/ntsc-rs.plugin"
```

Comment:
Might be nice to make `"/Library/Application Support/Adobe/Common/Plug-ins/7.0/MediaCore/ntsc-rs.plugin"` a variable. I may also check if the `--install-location` is passed in as an environment variable--I can't find any official documentation on these scripts, annoyingly.